src/llscan.cc

```diff
@@ -397,11 +417,50 @@ bool FindReferencesCmd::DoExecute(SBDebugger d, char** cmd,
     }
   }
 
+  delete scanner;
+
   result.SetStatus(eReturnStatusSuccessFinishResult);
   return true;
 }
 
 
+char** FindReferencesCmd::ParseScanOptions(char** cmd, ScanType& type) {
+  static struct option opts[] = {{"value", no_argument, nullptr, 'v'},
+                                 {"name", no_argument, nullptr, 'n'},
+                                 {nullptr, 0, nullptr, 0}};
+
+  int argc = 0;
+  for (char** p = cmd; p != nullptr && *p != nullptr; p++) argc++;
+
+  optind = 0;
+  opterr = 1;
+
+  bool found_scan_type = false;
+
+  do {
+    int arg = getopt_long(argc, cmd - 1, "vn", opts, nullptr);
+    if (arg == -1) break;
+
```

Comment:
Ok. It can drop straight out of the loop if it's found the option.
I've also dropped in changes to it matches the fix to the option parsing code under the other branch.
(The loop is slightly overkill until I add the option to limit the amount of output displayed but that should change shortly and in the meantime it looks like our other options parsing code.)